src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -372,20 +405,35 @@ public final void testExecuteIgnoredModules() throws IOException {
 
         final List<String> output = FileUtils.readLines(outputFile, StandardCharsets.UTF_8);
         final String errorMessage = "Content of file with violations differs from expected";
-        assertThat(errorMessage, output.get(0), is(auditStartedMessage.getViolation()));
-        assertThat(errorMessage, output.get(1), allOf(
-                startsWith("[WARN]"),
-                containsString("InputCheckstyleAntTaskError.java:4: "),
-                endsWith("@incomplete=Some javadoc [WriteTag]")));
-        assertThat(errorMessage, output.get(2), allOf(
-                startsWith("[ERROR]"),
-                endsWith("InputCheckstyleAntTaskError.java:7: "
-                    + "Line is longer than 70 characters (found 80). [LineLength]")));
-        assertThat(errorMessage, output.get(3), allOf(
-                startsWith("[ERROR]"),
-                endsWith("InputCheckstyleAntTaskError.java:9: "
-                    + "Line is longer than 70 characters (found 81). [LineLength]")));
-        assertThat(errorMessage, output.get(4), is(auditFinishedMessage.getViolation()));
+        assertWithMessage(errorMessage)
+                .that(output.get(0))
+                .isEqualTo(auditStartedMessage.getViolation());
+        assertWithMessage(errorMessage)
+                .that(output.get(1))
+                .startsWith("[WARN]");
+        assertWithMessage(errorMessage)
+                .that(output.get(1))
+                .contains("InputCheckstyleAntTaskError.java:4: ");
+        assertWithMessage(errorMessage)
+                .that(output.get(1))
+                .endsWith("@incomplete=Some javadoc [WriteTag]");
```

Comment:
Not sure about this. Please suggest.

Reply:
This can be combined into one assert,
```java
        assertWithMessage(errorMessage)
                .that(output.get(1))
                .startsWith("[WARN]");
        assertWithMessage(errorMessage)
                .that(output.get(1))
                .contains("InputCheckstyleAntTaskError.java:4: ");
```
can be simple
```java
        assertWithMessage(errorMessage)
                .that(output.get(1))
                .startsWith("[WARN]")
                .contains("InputCheckstyleAntTaskError.java:4: ");
```
Same for other.

Reply:
What of we combine `startsWith`, `contains` and `endsWith` assertions using regex expression?

```
assertWithMessage(errorMessage)
                .that(output.get(1))
                .matches("^[WARN].*InputCheckstyleAntTaskError.java:4: .*@incomplete=Some javadoc [WriteTag]$");
```
@pbludov what do you think?
@shashwatj07 could you validate approach?

Reply:
Okay.
Won't that hamper fluency?

Reply:
>  Won't that hamper fluency?

I think that regular expressions are more easy to read then a bunch of method invocations.

Reply:
Created issue in truth repository https://github.com/google/truth/issues/884

Reply:
@shashwatj07 lets use regexps for now

Reply:
done